### PR TITLE
Finished task 8 - posting a comment

### PR DESCRIPTION
**Pull request by Clatherine**
Completed posting a comment task. Please note, I also need task 7 reviewing as there was a problem with my pull request for that task, so if you could take a look at both that would be amazing, thank you!

### CODE REVIEW

File: src/Components/ArticleByIdPageComponents/LeaveComment.jsx [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
 
-    const handleSubmit =(event)=>{
-        event.preventDefault()
-        console.log('form submitted')
-        console.log(event.target)
-    }
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComment(article_id, mystery.user.username, commentBody).then((res) => {
+      console.log(res, "res");
```
Comment (nc-modou) [inline src/Components/ArticleByIdPageComponents/LeaveComment.jsx:15]: Remember to remove console logs once you're done debugging

File: src/Components/ArticleByIdPageComponents/LeaveComment.jsx [hunk 2]
```diff
@@ -1,30 +1,45 @@
...
+        Add comment:{" "}
+      </label>
+      <textarea
+        onChange={handleChange}
+        id="comment-body"
+        type="text"
+        name="comment-body"
+        value={commentBody}
+        required
+      />
```
Comment (nc-modou) [inline src/Components/ArticleByIdPageComponents/LeaveComment.jsx:39]: Good controlled component!

**Review by nc-modou — COMMENTED** [review]
Great work so far Catherine!

At the moment, when you click the button multiple times, you can spam the same comment over and over again because the button isn't disabled in any way. Have a look into this to stop users from making multiple comments while they wait for one to go through.

Also, there's no feedback on a failed comment post which is an important aspect to have.

File: src/Components/ArticleByIdPageComponents/LeaveComment.jsx (hunk 1 above)
Comment (Clatherine) [reply src/Components/ArticleByIdPageComponents/LeaveComment.jsx:15]: done!

File: src/Components/ArticleByIdPageComponents/LeaveComment.jsx (hunk 2 above)
Comment (Clatherine) [reply src/Components/ArticleByIdPageComponents/LeaveComment.jsx:39]: thank you!

**Comment by Clatherine** [comment]
Have updated code to prevent comment spamming through button disabling

**Comment by Clatherine** [comment]
Have added feedback comment for failed comments

**Pull request merged**